transformer_engine/pytorch/tensor/float8_tensor.py

```diff
@@ -715,7 +715,7 @@ def __torch_dispatch__(cls, func, types, args, kwargs=None):
                 )
             # deep copy the scale inverse tensor and quantizer as well.
             scale_inv = tensor._scale_inv.detach().clone()
-            quantizer = tensor._quantizer.copy()
+            quantizer = tensor._quantizer
```

Comment:
We should make sure comments are accurate:
```suggestion
            scale_inv = tensor._scale_inv.detach().clone()
            quantizer = tensor._quantizer  # Deep-copied within constructor
```